tests/unit/tracing/test_tracker.py

```diff
@@ -1,9 +1,9 @@
 """Unit test for SortTracker."""
 
 import pytest
-from vendor.sort import sort
 
 from tracing.tracker import BBox, Detection, SortTracker
+from vendor.sort import sort
```

Comment:
I feel perhaps this is wrong?  sort is an external library.

Reply:
not sure I understand what should be wrong?

Reply:
IMHO, sort is external and should be grouped with pytest.

Reply:
https://pycqa.github.io/isort/docs/configuration/options#known-third-party

Reply:
Fixed.